starknet_py/utils/schema.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from marshmallow import EXCLUDE, RAISE
+from marshmallow import Schema as SchemaToCopy
+
+print(os.environ.get("marshmallow_raise"))
+
+
+class Schema(SchemaToCopy):
+    class Meta:
+        unknown = RAISE if os.environ.get("marshmallow_raise") else EXCLUDE
```

Comment:
```suggestion
        unknown = RAISE if os.environ.get("STARKNET_PY_MARSHMALLOW_META_MODE") else EXCLUDE
```
or something similar. env variables should always be capital letters, and it may make sense to make it more descriptive + starknet.py specific